File: testdata/bench/go/BenchmarkTest00017.go
package bench

import (
	"database/sql"
	"net/http"
)

// Parameterized with channel
func Handler00017(w http.ResponseWriter, r *http.Request) {
	param := r.FormValue("id")
	ch := make(chan string, 1)
	ch <- param
	val := <-ch
	db, _ := sql.Open("sqlite3", ":memory:")
	db.Query("SELECT * FROM users WHERE id = ?", val)
}

File: testdata/bench/swift/BenchmarkTest00017.swift
import Vapor
import SQLite3

func handler017(_ req: Request) throws -> String {
    let query = "SELECT COUNT(*) FROM users WHERE active = 1"
    var db: OpaquePointer?
    sqlite3_open(":memory:", &db)
    sqlite3_exec(db, query, nil, nil, nil)
    return "counted"
}
